### DISCUSSION TITLE
Is Initializable a library or a preset?

### DISCUSSION THREAD

**Discussion by tevrat-aksoy** [OpenZeppelin/cairo-contracts#261] [Q&A] [answered]
Hello,

initializable has two external functions. As far as I understand, libraries should not have external functions. 
Is this initializable module supposed to be a library or  am I getting this wrong?

I am just getting used to cairo sorry if I am wasting your time.

**Comment by martriay** [comment, marked as answer]
Thanks for raising this! Indeed, that's a rule of libraries. Anyway, `Initializable` is not a library but a mock contract used only in tests. We should move it to the `test/` directory.

**Comment by martriay** [comment]
Created #263, closing this :)
